src/components/GraphiQL.js

```diff
@@ -495,6 +500,7 @@ export class GraphiQL extends React.Component {
         const fetch2 = observableToPromise(
           fetcher({
             query: introspectionQuerySansSubscriptions,
+            operationName: 'IntrospectionQuery',
```

Comment:
Same here but in this case, it can be hardcoded as `introspectionQuerySansSubscriptionsName` in `introspectionQueries.js`.